src/CliModels.ts

```diff
@@ -26,14 +27,20 @@ export class CliInput {
     return this._options;
   }
 
-  get helpMessage(): HelpMessage {
-    return this._helpMessage;
+
+  get arguments(): Arguments {
+    return this._arguments;
   }
 
   static createFromMeow(meowOutput: any): CliInput {
-    const [command] = meowOutput.input;
-    const help = meowOutput.help;
+    const [command, ...args] = meowOutput.input;
     const { context, watch } = meowOutput.flags;
-    return new CliInput(command || 'help', { context, watch }, help);
+    return new CliInput(command || 'help', { context, watch }, args);
+  }
+
+  static createSubCommand(cliInput: CliInput): CliInput {
+    const [command, ...args] = cliInput.arguments;
+    return new CliInput(command || "help", cliInput.options, args);
```

Comment:
there is no such thing as `help` command, right? so even if no command is passed, nothing will really happen, help will not show up really

Reply:
Yeah, but as I was reusing the `CliInput` class and it needs a string that's why passed the help command, it actually does nothing as help text si handled by `meow`

Reply:
feel wierd, let us leave it as it is, just use `'` instead of `"` and please create a followup issue to clear that out, maybe we actually should have `help` as command and not flag 🤔